plugins/com.google.cloud.tools.eclipse.appengine.ui/src/com/google/cloud/tools/eclipse/appengine/ui/messages.properties

```diff
@@ -13,4 +13,7 @@ cloud.sdk.not.installed=Could not find the Google Cloud SDK
 cloud.sdk.out.of.date=Cloud SDK too old
 appengine.java.component.missing=App Engine Java Component is missing
 appengine.java.component.not.installed=The Cloud SDK App Engine Java component is not installed
-appengine.libraries.group=Libraries to add to build path
\ No newline at end of file
+appengine.libraries.group=Libraries to add to build path
+objectify.tooltip=com.googlecode.objectify, a Java data access API designed for Google Cloud Datastore
```

Comment:
Put `com.googlecode.objectify` at the end with in parentheses (like the `endpoints.tooltip` below)?